### ISSUE TITLE
Passing `{ Force: true }` does not prevent a detached check on `click`, `type`

### ISSUE THREAD

**Issue by jonathonherbert** [cypress-io/cypress#16397]
### Current behavior

As with #9523, when calling something actionable on a paragraph element in [Prosemirror](https://prosemirror.net/), we get e.g. `cy.type() failed because this element is detached from the DOM.`

This is understandable, but it also happens when `{ force: true }` is provided as an option to the actionable. [The docs suggest that an actionable check should not occur when this is passed](https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Forcing), but this does not appear to be the case.

This only appears to happen during `open`. When we call `cypress run` for the same test, it passes.

`click({ force: true })` exhibits the same behaviour. Other actions are untested.

<img width="840" alt="Screenshot 2021-05-07 at 17 19 42" src="https://user-images.githubusercontent.com/7767575/117479601-6e73d580-af58-11eb-9ed2-9d40a4e4ed91.png">

### Desired behavior

No error is thrown, and the action occurs on the target element.

### Test code to reproduce

Example test:

```js
describe('page', () => {
  it("should get element", function () {
    cy.visit("https://cypress-prosemirror-bug.glitch.me/");
    cy.get(".ProseMirror p").type("Edited text", { force: true });
  });
})
```

Repo [here](https://github.com/jonathonherbert/cypress-test-tiny).

### Versions

Mojave 10.14.3, Cypress 7.2.0.716687, Chrome 90.

**Comment by jennifer-shehane** [comment]
I think you need to type into the contenteditable element itself to have proper typing with how Cypress works now, but I can see how this doesn't give much control over where typing occurs. 

```js
it("should get element", function () {
  cy.visit("https://cypress-prosemirror-bug.glitch.me/");
  cy.get(".ProseMirror").type("Edited text");
});
```

I would suggest checking this thread: https://github.com/cypress-io/cypress/issues/7306 People have different workarounds for the detached DOM issue, sending force is not a gauranteed workaround for the detached DOM check.

Duplicate of https://github.com/cypress-io/cypress/issues/7306

**Comment by jonathonherbert** [comment]
Thanks for the swift reply! Typing into the .Prosemirror element works as expected in this case.

For completeness, I'm not convinced that this is a duplicate of #7306, although that issue might be a solution to this problem. If I understand correctly, in this case `{ force: true }` is not being respected, and the actionable check is happening when it shouldn't – which would be a separate issue.

Regardless, I'm unblocked – so many thanks for the help.

**Comment by jennifer-shehane** [comment]
It does seem like the docs and the code indicate that the check for detachment won't happen if you pass force: true. But, thinking about it, I'm not sure how that's possible since it's lost reference to the original element since it's detached.
